server/src/com/cloud/api/ApiResponseGsonHelper.java

```diff
@@ -27,30 +29,40 @@
 import com.google.gson.GsonBuilder;
 
 /**
- * The ApiResonseGsonHelper is different from ApiGsonHelper - it registeres one more adapter for String type required for api response encoding
+ * The ApiResonseGsonHelper is different from ApiGsonHelper - it registers one more adapter for String type required for api response encoding
```

Comment:
We should upgrade gson before further expanding on this feature of it. Exlusion strategies have somewhat changed in version 2 in the sense that exclusion is cahced inside gson and a change in loglevel is not endorsed.